.github/workflows/kunit.yml

```diff
@@ -49,4 +49,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --target ${{ matrix.target }} --verbose
+          args: --target ${{ matrix.target }} --release --verbose
```

Comment:
Why release here?

Reply:
Because ``page_alloc_out_of_memory`` test is really slow without it (more than a minute)